purchases/build.gradle

```diff
@@ -15,14 +15,13 @@ android {
         buildConfig true
     }
 
-    productFlavors {
-        integrationTest {
-            testApplicationId obtainTestApplicationId()
-            testBuildType obtainTestBuildType()
-            packagingOptions {
-                resources.excludes.add("META-INF/LICENSE.md")
-                resources.excludes.add("META-INF/LICENSE-notice.md")
-            }
+    defaultConfig {
+        testApplicationId obtainTestApplicationId()
+        testBuildType obtainTestBuildType()
+        // TODO: Revise this
```

Comment:
I'm not sure if there is a better way to do this, but not sure how to specify packagingOptions only for tests...